front/components/markdown/PastedAttachmentBlock.tsx

```diff
@@ -0,0 +1,22 @@
+import { AttachmentChip } from "@dust-tt/sparkle";
+import { PaperclipIcon } from "lucide-react";
+import React from "react";
+import { visit } from "unist-util-visit";
+
+export function PastedAttachmentBlock({ title }: { title: string }) {
+  return <AttachmentChip label={title} icon={PaperclipIcon} />;
+}
+
+export function pastedAttachmentDirective() {
+  return (tree: any) => {
```

Comment:
Any chance we can be stricter than `any` here?

Reply:
Could not find how to avoid this any sorry :(
any is used in simlar places here: 

https://github.com/dust-tt/dust/blob/bba9f9d91036bd6c8dddc62700c8e893ed255072/front/components/markdown/ContentNodeMentionBlock.tsx#L16

https://github.com/dust-tt/dust/blob/bba9f9d91036bd6c8dddc62700c8e893ed255072/front/components/markdown/Image.tsx#L52

https://github.com/dust-tt/dust/blob/bba9f9d91036bd6c8dddc62700c8e893ed255072/front/components/markdown/VisualizationBlock.tsx#L88